.github/workflows/upgrade-subctl.yml

```diff
@@ -0,0 +1,45 @@
+---
+name: Upgrade command
+
+on:
+  pull_request:
+    types: [ready_for_review, opened, reopened, synchronize, converted_to_draft, labeled]
```

Comment:
Any specific reason why we want to run this job when a PR is converted to draft?

Reply:
No, I suspect this was copied from `upgrade-e2e.yml` where it is paired with an `if` clause in the job.